test/e2e/lib/gpg.bash

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+function tmp_gnupghome() {
+  local tmp_home
+  tmp_home=$(mktemp -d)
+  export GNUPGHOME="$tmp_home"
```

Comment:
I find this side-effect confusing. No strong opinion, but I wonder whether it's worth having this function instead of inlining it